spanner/testdata/book.pb.go

```diff
@@ -0,0 +1,229 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.21.5
+// source: book.proto
+
+package testdata
+
+import (
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type Genre int32
+
+const (
+	Genre_BLUES     Genre = 0
+	Genre_CLASSICAL Genre = 1
+	Genre_COUNTRY   Genre = 2
+	Genre_ROCK      Genre = 3
+)
+
+// Enum value maps for Genre.
+var (
+	Genre_name = map[int32]string{
+		0: "BLUES",
+		1: "CLASSICAL",
+		2: "COUNTRY",
+		3: "ROCK",
+	}
+	Genre_value = map[string]int32{
+		"BLUES":     0,
+		"CLASSICAL": 1,
+		"COUNTRY":   2,
+		"ROCK":      3,
+	}
+)
+
+func (x Genre) Enum() *Genre {
+	p := new(Genre)
+	*p = x
+	return p
+}
+
+func (x Genre) String() string {
+	return protoimpl.X.EnumStringOf(x.Descriptor(), protoreflect.EnumNumber(x))
+}
+
+func (Genre) Descriptor() protoreflect.EnumDescriptor {
+	return file_book_proto_enumTypes[0].Descriptor()
+}
+
+func (Genre) Type() protoreflect.EnumType {
+	return &file_book_proto_enumTypes[0]
+}
+
+func (x Genre) Number() protoreflect.EnumNumber {
+	return protoreflect.EnumNumber(x)
+}
+
+// Deprecated: Use Genre.Descriptor instead.
+func (Genre) EnumDescriptor() ([]byte, []int) {
+	return file_book_proto_rawDescGZIP(), []int{0}
+}
+
+type Book struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Isbn   int64  `protobuf:"varint,1,opt,name=isbn,proto3" json:"isbn,omitempty"`
+	Title  string `protobuf:"bytes,2,opt,name=title,proto3" json:"title,omitempty"`
+	Author string `protobuf:"bytes,3,opt,name=author,proto3" json:"author,omitempty"`
+	Genre  Genre  `protobuf:"varint,4,opt,name=genre,proto3,enum=main.Genre" json:"genre,omitempty"`
+}
+
+func (x *Book) Reset() {
+	*x = Book{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_book_proto_msgTypes[0]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *Book) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*Book) ProtoMessage() {}
+
+func (x *Book) ProtoReflect() protoreflect.Message {
+	mi := &file_book_proto_msgTypes[0]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use Book.ProtoReflect.Descriptor instead.
+func (*Book) Descriptor() ([]byte, []int) {
```

Comment:
Please replace book example with the standard Singers/Albums one.

Reply:
Replaced `Book` example with standard `Singers` proto.